projects/js-packages/charts/src/components/legend/base-legend.tsx

```diff
@@ -72,18 +74,35 @@ export const BaseLegend: FC< LegendProps > = ( {
 							flexDirection={ itemDirection }
 							{ ...legendItemProps }
 						>
-							<LegendShape
-								shape={ shape }
-								height={ shapeHeight }
-								width={ shapeWidth }
-								margin={ shapeMargin }
-								item={ domain[ i ] }
-								itemIndex={ i }
-								label={ label }
-								fill={ fill }
-								size={ size }
-								shapeStyle={ getShapeStyle }
-							/>
+							{ renderGlyph ? (
```

Comment:
Right now, I have made the usual square/circle, etc shape vs the glyph optional cc @kangzj, would it be expected to render both at times? https://linear.app/a8c/issue/CHARTS-16#comment-8a7bab66